BP/common/landed_titles/BP_empire_roman.txt

```diff
@@ -0,0 +1,3859 @@
+						romano_british = cn_forum_livii
+						dalmatian = cn_forum_livii
+						african_romance = cn_forum_livii
+						romano_gothic = cn_forum_livii
+						romano_dacian = cn_forum_livii 
+						romano_frank = cn_forum_livii 
+						romano_aramean = cn_forum_livii 
+						romano_vandalic = cn_forum_livii 
+						thraco_roman = cn_forum_livii 
+						illyro_roman = cn_forum_livii 
+						raetoroman = cn_forum_livii 
+						noricroman = cn_forum_livii 
+						pannonian = cn_forum_livii 
+					}
+					color = { 229 165 115 }
+					color2 = { 255 255 255 }
+				}
+				b_rimini = {
+					province = 2547
+					cultural_names = {
+						#Roman
+						roman = cn_ariminum
+						romano_gallic = cn_ariminum
+						romano_hispanic = cn_ariminum
+						romano_british = cn_ariminum
+						dalmatian = cn_ariminum
+						african_romance = cn_ariminum
+						romano_gothic = cn_ariminum
+						romano_dacian = cn_ariminum 
+						romano_frank = cn_ariminum 
+						romano_aramean = cn_ariminum 
+						romano_vandalic = cn_ariminum 
+						thraco_roman = cn_ariminum 
+						illyro_roman = cn_ariminum 
+						raetoroman = cn_ariminum 
+						noricroman = cn_ariminum 
+						pannonian = cn_ariminum 
+					}
+					color = { 229 165 115 }
+					color2 = { 255 255 255 }
+				}
+			}
+			c_bologna = {
+				color = { 209 145 95 }
+				color2 = { 255 255 255 }
+				
+				cultural_names = {
+					#Roman
+					roman = cn_bononia
+					romano_gallic = cn_bononia
+					romano_hispanic = cn_bononia
+					romano_british = cn_bononia
+					dalmatian = cn_bononia
+					african_romance = cn_bononia
+					romano_gothic = cn_bononia
+					romano_dacian = cn_bononia 
+					romano_frank = cn_bononia 
+					romano_aramean = cn_bononia 
+					romano_vandalic = cn_bononia 
+					thraco_roman = cn_bononia 
+					illyro_roman = cn_bononia 
+					raetoroman = cn_bononia 
+					noricroman = cn_bononia 
+					pannonian = cn_bononia 
+					#Germanic
+					skirian = cn_bononia
+					ostrogothic = cn_bononia
+					thuringian = cn_bononia
+				}
+				
+				b_bologna = {
+					province = 2542
+					cultural_names = {
+						#Roman
+						roman = cn_bononia
+						romano_gallic = cn_bononia
+						romano_hispanic = cn_bononia
+						romano_british = cn_bononia
+						dalmatian = cn_bononia
+						african_romance = cn_bononia
+						romano_gothic = cn_bononia
+						romano_dacian = cn_bononia 
+						romano_frank = cn_bononia 
+						romano_aramean = cn_bononia 
+						romano_vandalic = cn_bononia 
+						thraco_roman = cn_bononia 
+						illyro_roman = cn_bononia 
+						raetoroman = cn_bononia 
+						noricroman = cn_bononia 
+						pannonian = cn_bononia 
+						#Germanic
+						skirian = cn_bononia
+						ostrogothic = cn_bononia
+						thuringian = cn_bononia
+					}
+					color = { 209 145 95 }
+					color2 = { 255 255 255 }
+				}
+				b_cona = {
+					province = 2529
+					color = { 209 145 95 }
+					color2 = { 255 255 255 }
+				}
+				b_monte_sole = {
```

Comment:
This can be Romanised if necessary, with a comment in case a better option exists